### PR TITLE
About modal

### LINKED ISSUE #122: About modal (https://github.com/dcgtc/dgrants/blob/richard-ui3/app/src/components/About.vue)
Dependent on #116

### PR DESCRIPTION
**Pull request by carletex**
Fixes #122 

WIP until #131 is merged.

- **Build number**: will be implemented in #132 
-  **Contributors**: I requested access to this repo for the [all-contributors app](https://allcontributors.org/). I created a test json file for testing.

### CODE REVIEW

File: app/src/components/About.vue
Comment (thelostone-mc) [inline app/src/components/About.vue:89]: wouldn't this a view instead of a component ?

Comment (carletex) [reply app/src/components/About.vue:89]: I think views are meant for "routed pages", right? Since this is going to be a modal loading on top of any page, I think it should be a component. But I might be wrong!

Comment (gdixon) [reply app/src/components/About.vue:89]: I'd agree that this should be a component 👍

Comment (mds1) [reply app/src/components/About.vue:89]: > I think views are meant for "routed pages", right? Since this is going to be a modal loading on top of any page, I think it should be a component.
Agreed/correct!

**Comment by carletex** [comment]
@phutchins I still can't configure the all-contributor app:

image

Can you install it? Maybe I'll be able to configure it after you installed it. https://github.com/apps/allcontributors/installations/new

**Comment by carletex** [comment]
Hey @apbendi 

Just fixed the smushing (TIL a new word ^^) adding a provisional centering to the page main content (before the centering was placed in the whole #app, that's why I initially removed it)

Question about (rebase & ) merging PRs workflow: Should I do it once the PR is approved? Or it's something you prefer to do yourself?

**Comment by apbendi** [comment]
Hey @carletex, once approved by one ScopeLift and one Gitcoin reviewer, it can be merged by anyone, as long as it is rebased and fast forward only. Looks like we need one Gitcoin reviewer here.

**Review by gdixon — APPROVED** [review]
LGTM 🚀 Nice job @carletex!

**Pull request merged**
